x-pack/legacy/plugins/siem/public/components/ml_popover/api.tsx

```diff
@@ -189,7 +178,6 @@ export const stopDatafeeds = async ({
       'kbn-system-api': 'true',
       'content-type': 'application/json',
       'kbn-xsrf': kbnVersion,
```

Comment:
No need to add now (as it's been missed before), but looks like these two API calls are the only ones missing the `'kbn-version': kbnVersion` header. We're fine with just `'kbn-xsrf', but should probably be there for consistency with the others. Can refactor remainder in the next NP PR when this all changes again anyway.

Reply:
Maybe we should just create a generic function to build these headers everywhere.